apps/daimo-mobile/src/view/sheet/OnboardingChecklistBottomSheet.tsx

```diff
@@ -0,0 +1,138 @@
+import Octicons from "@expo/vector-icons/Octicons";
+import { TouchableHighlight } from "@gorhom/bottom-sheet";
+import { StyleSheet, View } from "react-native";
+
+import { useOnboardingChecklist } from "../../logic/onboarding";
+import { Account } from "../../model/account";
+import { ButtonBig } from "../shared/Button";
+import Spacer from "../shared/Spacer";
+import { color, touchHighlightUnderlay } from "../shared/style";
+import { DaimoText, TextBody, TextH3 } from "../shared/text";
+import { useWithAccount } from "../shared/withAccount";
+
+export function OnboardingChecklistBottomSheet() {
+  const Inner = useWithAccount(OnboardingChecklistBottomSheetInner);
+  return <Inner />;
+}
+
+function OnboardingChecklistBottomSheetInner({
+  account,
+}: {
+  account: Account;
+}) {
+  const {
+    hasBackup,
+    farcasterConnected,
+    handleSecureAccount,
+    handleConnectFarcaster,
+    dismissSheet,
+  } = useOnboardingChecklist(account);
+
+  return (
+    <View>
+      <TextH3 style={{ textAlign: "center" }}>Onboarding checklist</TextH3>
+      <Spacer h={16} />
+      <TextBody style={{ textAlign: "center", color: color.grayMid }}>
+        Complete these items to finish account setup
+      </TextBody>
+      <Spacer h={24} />
+      <View style={styles.separator} />
+      <ChecklistRow
+        step={1}
+        title="Secure your account"
+        description="Add a passkey backup to your account"
+        onPress={handleSecureAccount}
+        done={hasBackup}
+      />
+      <ChecklistRow
+        step={2}
+        title="Connect Farcaster"
+        description="Import your profile image and connections"
+        onPress={handleConnectFarcaster}
+        done={farcasterConnected}
+      />
+      <Spacer h={24} />
+      <View style={{ paddingHorizontal: 24 }}>
+        <ButtonBig
+          type="subtle"
+          title={`I'll get to it later`}
+          onPress={dismissSheet}
+        />
+      </View>
+    </View>
+  );
+}
+
+function ChecklistRow({
+  step,
+  title,
+  description,
+  onPress,
+  done,
+}: {
+  step: number;
+  title: string;
+  description: string;
+  onPress(): void;
+  done: boolean;
+}) {
+  return (
+    <TouchableHighlight onPress={onPress} {...touchHighlightUnderlay.subtle}>
+      <View>
+        <View style={styles.row}>
+          <View style={{ flexDirection: "row", alignItems: "center" }}>
+            <View
```

Comment:
recommend making this lest nested or pulling out a component for clarity

vs `<View><View>`x4